src/lib/ui/thermostat/TemperatureControlGroup.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box } from '@mui/material'
+import { useArgs } from '@storybook/preview-api'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { TemperatureControlGroup } from './TemperatureControlGroup.js'
+
+const meta: Meta<typeof TemperatureControlGroup> = {
+  title: 'Library/TemperatureControlGroup',
+  tags: ['autodocs'],
+  component: TemperatureControlGroup,
+}
+
+type Story = StoryObj<typeof TemperatureControlGroup>
+
+export const Content: Story = {
+  render: (props) => {
+    const [_, setArgs] = useArgs()
```

Comment:
We can update all the stories to use args consistently, so out of scope for this PR, but fwiw I think the intended way to define arg defaults is via argtypes shown here https://storybook.js.org/docs/react/api/arg-types#argtypes

Otherwise the control will be out of sync with the initial value.

Reply:
Ah nice! I was having trouble finding docs on this.